target_postgres/tests/test_sdk.py

```diff
@@ -0,0 +1,77 @@
+""" SDK tests for target postgres """
+# flake8: noqa
+import pytest
+from singer_sdk.testing import get_target_test_class
+from singer_sdk.testing.suites import TestSuite
+from singer_sdk.testing.target_tests import (
```

Comment:
Dang I was hoping we could do an exclude instead of an explicit include so tests added to the SDK would auto add here. Hmm I'll put an issue into the SDK and see what they think